administrator-manual/en/proxy_pass.rst

```diff
@@ -1,20 +1,33 @@
+.. _proxy_pass-section:
+
 =============
 Reverse proxy
 =============
 
 The :index:`reverse proxy` feature is useful when you want to access internal sites
 from the outside network.
 
-Typical scenario:
+Path and virtual host rules
+===========================
+
+A web client request can be forwarded to another web server transparently,
+according to two types of matching rules:
+
+* Requests matching an URL path, like ``http://mydomain.com/mysite``
+* Requests matching a virtual host name, like ``http://my.secondary-domain.com``
+
+We will see how to set up the two types of rule with a couple of examples.
```

Comment:
We never used such discursive form inside the manual.
I'd remove the entire phrase.